src/arrayMethodIncludes.js

```diff
@@ -5,8 +5,14 @@
  *
  */
 function applyCustomIncludes() {
-  [].__proto__.includes2 = function() {
-    // write code here
+  [].__proto__.includes2 = function(item = null, index = 0) {
+    let result = false;
+    for (let i = index; i < this.length; i++) {
+      if (this[i] === item || (isNaN(item) && isNaN(this[i]))) {
+        result = true;
```

Comment:
Almost great, but one improvement. return here and `result ` will be not needed anymore, just return false in the end